status-im/status-react
# [FIX #4347] Update Launch Screens

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

Fixes #4347

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
Updates launch screens to those specified in #4347 

### Steps to test:
- Open Status
- See if launch screen reflects new assets



status: ready
